src/runmode-unix-socket.c

```diff
@@ -389,6 +518,29 @@ void UnixSocketPcapFile(TmEcode tm)
 #endif
 }
 
+void UnixSocketPcapFolder(TmEcode tm)
+{
+#ifdef BUILD_UNIX_SOCKET
+    switch (tm) {
+        case TM_ECODE_DONE:
+            unix_manager_folder_task_running = 0;
+            break;
+        case TM_ECODE_FAILED:
+            unix_manager_folder_task_running = 0;
+            unix_manager_folder_task_failed = 1;
+            break;
+        case TM_ECODE_OK:
+            unix_manager_folder_task_running = 1;
+            break;
+    }
+#endif
+}
+
+void UnixSocketPcapFolderSetLastProcessed(time_t time)
```

Comment:
minor: 'time' is the name of a common function, I prefer to avoid naming clashes/confusion